READMEs/setup-local.md

```diff
@@ -52,9 +52,8 @@ cd my_project
 source venv/bin/activate
```

Comment:
I wonder if we want to have these other changes that remove Mac & Win.

When barge gets fixed, them we'd have to re-insert them. I worry that it will be error-prone to reinsert everything.

Two options:
1. Don't make these changes
2. Keep these changes, but also keep track of possible things to re-insert once Mac/Win are working on local

I'm on the fence on this, ie I'm fine either way.
I'm a bit on the fence on this.

Reply:
I suggest to re-insert the changes later as I will be filming the local setup again on a Mac, so I will notice then the instructions need clarity for MacOS + Windows at that point in time.

Reply:
OK. Please go ahead with that:)